website/photos/templates/photos/album.html

```diff
@@ -12,18 +12,21 @@
             <h1 class="section-title">
                 {{ album.title }}
                 {% if album.event %}
-                <a href="{{ album.event.get_absolute_url }}"
-                   class="btn btn-primary btn-first"
-                   data-bs-toggle="tooltip"
-                   data-bs-placement="top"
-                   title=""
-                   data-original-title="{% trans "Event" %}">
-                    <i class="fas fa-calendar"></i>
-                </a>
+                    <a href="{{ album.event.get_absolute_url }}"
+                       style="top: auto"
+                       class="btn btn-primary btn-first"
+                       data-bs-toggle="tooltip"
+                       data-bs-placement="top"
+                       title=""
+                       data-original-title="{% trans "Event" %}">
+                        <i class="fas fa-calendar"></i>
+                    </a>
+                    <span style="position: absolute; right: 50px">{{ album.date|date:"d-m-Y" }}</span>
```

Comment:
If we would style every element with inline CSS it would get messy very quickly. Aka: this should be in a SCSS file :)

Reply:
Fair point. Current solution should be a lot less messy and no longer requires hard coding things like `btn-first`! Let me know what you think